base/src/consensus_state.rs

```diff
@@ -18,6 +20,11 @@ use smart_default::SmartDefault;
 #[derive(Clone, Default, PartialEq, Debug, From, Serialize)]
 pub struct VrfOutputTruncated(pub Vec<u8>);
 
+impl_ref_hashable!(
+    VrfOutputTruncated,
+    append_bytes => 0
```

Comment:
This macro is cute but I think it makes things less readable rather than more. It took me a while to figure out what is going on in this one for example.

Also it doesn't really save much space. I think this would be better without it

Reply:
removed macro use for now